test/integration/acceptance/grant_fixture.hpp

```diff
@@ -0,0 +1,411 @@
+/**
+ * Copyright Soramitsu Co., Ltd. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#ifndef IROHA_GRANT_FIXTURE_H
+#define IROHA_GRANT_FIXTURE_H
+
+#include <gtest/gtest.h>
+
+#include "builders/protobuf/queries.hpp"
+#include "builders/protobuf/transaction.hpp"
+#include "framework/integration_framework/integration_test_framework.hpp"
+#include "framework/specified_visitor.hpp"
+#include "integration/acceptance/acceptance_fixture.hpp"
+
+using namespace integration_framework;
+
+using namespace shared_model;
+using namespace shared_model::interface;
+using namespace shared_model::interface::permissions;
+
+class GrantPermissionFixture : public AcceptanceFixture {
+ public:
+  using TxBuilder = TestUnsignedTransactionBuilder;
+
+  /**
+   * Creates the transaction with the user creation commands
+   * @param perms are the permissions of the user
+   * @return built tx and a hash of its payload
+   */
```

Comment:
Please, add all parameters' descriptions

Reply:
Also, it's unclear from the comment's head, what this function does. Please, clarify, something like: "Creates transaction, which creates a user with specified permissions"